edifact/incoming/edifact_recep_consumer.py

```diff
@@ -0,0 +1,5 @@
+
+
+class EdifactRecepConsumer:
```

Comment:
Just spotted this, could these three mirror the outbound structure, so put the files in inbound/converter

These are mainly orchestration classes. The /edifact package contains internal representations of edifact message segments and is used by the whole application.

Reply:
We should probably do a breakdown of the package structure to explain what goes where and why...

Reply:
By three do you mean inbound sequence number manager, Edifact Recep Consumer and edifact recep producer?

Reply:
EdifactRecepConsumer, EdifactRecepProducer, EdifactToFhir